File: 9_PalindromeNumber/python/PalindromeNumber.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        str_num = str(x)
        left = 0;
        right = len(str_num)-1;
        while right>=left:
            if str_num[left] != str_num[right]:
                return False
            right-=1
            left+=1
        return True

File: 9_PalindromeNumber/c#/PalindromeNumber.cs
public class Solution {
    public bool IsPalindrome(int x) {
        string str_num = x.ToString();
        int left = 0;
        int right = str_num.Length-1;
        while(left<=right){
            if(str_num[left] != str_num[right]){
                return false;
            }
            left++;
            right--;
        }
        return true;
    }
}